h/accounts/schemas.py

```diff
@@ -189,6 +189,11 @@ class EditProfileSchema(CSRFSchema):
         default='',
         missing=colander.null
     )
+    emailAgain = colander.SchemaNode(
```

Comment:
> We might not need to do any of the validation ourselves if we just set the widget kwarg on the email node to one of these.

This results in a great big stack trace from deform.

I'll try to figure it out but to be honest I feel this is a ludicrously complicated way to compare two strings (involving two third-party libraries). How many forms do we actually have in h? Wondering whether having deform and colander in our code at all is justifiable.